channels/ru.m3u

```diff
@@ -369,16 +283,10 @@ https://strm.yandex.ru/kal/diving/diving0.m3u8
 http://strm.yandex.ru/kal/dialogi/dialogi0.m3u8
 #EXTINF:-1 tvg-id="Dozhd.ru" tvg-name="Дождь" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",Дождь [No audio]
 https://a3569457435-s78511.cdn.ngenix.net/open/TVRain_noaudio/playlist_DVR.m3u8
+#EXTINF:-1 tvg-id="Dozhd.ru" tvg-name="Дождь" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",Дождь [No audio]
```

Comment:
has audio

Reply:
Copied and pasted without getting the mention off indeed. Gonna change that.